Files/MultilingualResources/Files.hu-HU.xlf

```diff
@@ -2352,31 +2352,31 @@
         </trans-unit>
         <trans-unit id="SidebarFavorites" translate="yes" xml:space="preserve">
           <source>Favorites</source>
-          <target state="new">Favorites</target>
+          <target state="new">Kedvencek</target>
```

Comment:
All translated strings should be marked as `translated`.

Reply:
Oh, I was a little careless. Thank you for the warning!